contracts/0.4.24/oracle/LidoOracle.sol

```diff
@@ -101,31 +101,32 @@ contract LidoOracle is ILidoOracle, AragonApp {
     So we should take another popular metric to express this - PPM or 1e-6
     daily_granularity_in_ppm = 2.74e-05 / 1e-6 = 27 (pretty good)
     with 100% buffer for future spikes it will be 50.
+    Annualized 50 * 365 ~~ 100000 PPM
     May be changed by Gov.
     */
     bytes32 internal constant ALLOWED_BEACON_BALANCE_ANNUAL_RELATIVE_INCREASE_POSITION =
         keccak256("lido.LidoOracle.allowedBeaconBalanceAnnualRelativeIncrease");
-    uint256 internal constant DEFAULT_ALLOWED_BEACON_BALANCE_ANNUAL_RELATIVE_INCREASE = 50;  // PPM
+    uint256 public constant DEFAULT_ALLOWED_BEACON_BALANCE_ANNUAL_RELATIVE_INCREASE = 100000;  // PPM ~ 10%
 
     /*
     When slashing happens, the balance may decrease at a much faster pace.
     On the start the system will have about 100 validators and we can guess one of them can be slashed for any reason.
```

Comment:
```suggestion
    Slashing are one-time events that decrease the balance a fair amount - a few percent at a time in a realistic scenario. Thus, instead of sanity check for an APR, we check if the plain relative decrease is within bounds.
```

Reply:
done